README.md

```diff
@@ -117,6 +111,28 @@ We also support inline queries which then trigger messages in the goal panel.
 
 * Supports \_CoqProject
 
+### Since version 2.1.7
+
+* Outline
+
+We now support a document outline, which displays theorems and definitions in the document.
+
+![](gif/outline.gif)
+
+* Ellipsis for the goal panel
+
+Goals can now be ellided. First through the `"vscoq.proof.maxDepth"` setting which ellides a goal if the display becomes to large.
```

Comment:
Should this possibly be `"vscoq.goals.maxDepth"`? 

Also might I suggest adding a full description of that option to the "Goal and info view panel" section later in this README, as well as a description for the `"vscoq.proof.block"` in "Proof checking"

Reply:
On it !